less/home-page.less

```diff
@@ -1 +1,101 @@
-// Home Content
\ No newline at end of file
+// Home Content
+.home {
+    .earth {
+        display: flex;
+
+        img {
+            width: 100%;
+            padding-bottom: 20px;
```

Comment:
Did you consider having a ratio-sort of approach to padding? (eg. one variable, and then math functions based on that variable) Might be something to think about 🤔